deepview-explore/src/skyline_session.ts

```diff
@@ -142,17 +142,18 @@ export class SkylineSession {
     }
 
     connect() {
-        this.connection.connect(this.port, this.addr, this.on_open.bind(this));
+        this.connection.connect(this.port, this.addr);
```

Comment:
Is there a way for the connection status to update if the frontend reconnects? I think the callback function is need to make sure the frontend can reconnect. In this case, the callback should be this.on_connect instead of this,open

Reply:
I made a video to explain better the current situation and the proposed change.
https://drive.google.com/file/d/19cypooi60meDisYFNLh047a5oc3IZ3MI/view?usp=sharing
and this is a link that can be of help https://stackoverflow.com/questions/25791436/reconnect-net-socket-nodejs.